README.md

```diff
@@ -70,13 +70,22 @@ MyAttributedObject.new(
 SimpleFoo.new(bar: 12) == SimpleFoo.new(bar: 12)
 ```
 
+### Whitelist
+```ruby
+class MyAttributedObject
+  include AttributedObject::Strict
+
+  attribute :planet, :string, whitelist: ['Earth', 'Mars']
```

Comment:
imho combine the documentation with
disallow and also make naming similar. maybe `only_allow:` ?

https://github.com/mswiszcz/attributed_object/blob/06db193d1e2533fa38ca01682894ac321a88bae5/README.md#disallow-nil

Reply:
I actually disliked using only_allow, solely because disallow does not expect array but an exact value, that's why I went for "*list"

Reply:
yes i had the same thought.. just `only`? for some reason whitelist feels a bit out of place in this dsl.. no idea why. you decide

Reply:
I like whitelists and blacklists, I would keep it as it is